### PR TITLE
Testing: Use newly published jest-console package in test setup

### PR DESCRIPTION
**Pull request by gziolo**
## Description

`jest-console` test matchers were extracted to https://github.com/WordPress/packages to allow a wider application. This PR updates our setup to consume this newly published package.

## How Has This Been Tested?

`npm test` and Travis.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@wordpress/a11y": "0.1.0-beta.1",
     "@wordpress/hooks": "1.0.1",
+    "@wordpress/jest-console": "1.0.0",
```
Comment (youknowriad) [inline package.json]: I still think this should be just `@wordpress/jest` but not very important right now

**Review by youknowriad — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (gziolo) [reply package.json]: I know, I know :) We can discuss it further when we have more functionality to add there.

**Review by ntwb — APPROVED** [review]
👍

File: package.json (hunk 1 above)
Comment (aduth) [reply package.json]: For general usefulness, might having single-purpose modules here be preferred? vs. a monolithic utility library.

I don't feel strongly one way or the other, and it likely varies by the type of module (e.g. for unit tests, I'm more likely to pick and choose very specific utility support to add).

Comment (gziolo) [reply package.json]: My initial idea was to consolidate all very specific utilities under WordPress/scripts.

**Pull request merged**